test/Motor.Extensions.Hosting.Kafka_UnitTest/KafkaMessageTests.cs

```diff
@@ -105,7 +114,7 @@ private KafkaConsumerOptions<T> GetConsumerConfig<T>(string topic, string groupI
             Topic = topic,
             GroupId = groupId,
             CommitPeriod = 1,
-            BootstrapServers = $"{_fixture.Hostname}:{_fixture.Port}",
+            BootstrapServers = $"localhost:3000",
```

Comment:
I just realized that this is now a unit test and won't be using any Kafka Server at all. I assumed that you started Kafka locally for these tests as you changed this to `localhost`. I suggest to make this more clear by using a completely made up host name:

```suggestion
            BootstrapServers = $"willNotBeUsed:1234",
```